### PR TITLE
Update outdated changelog

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.4.0 - Release
+  - Improve error handling, group switching (Stretch).
```
Comment (bouwew) [inline CHANGELOG.md:4]: Stretch is wrong, must be Adam

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+## 1.3.0 - Only released in alpha/beta
+  - Initial support for Stretch (v2/v3) including tests
+  - Support gzip encoding
```
Comment (bouwew) [inline CHANGELOG.md:8]: Force gzip encoding, work-around for aiohttp-error

**Review by bouwew — COMMENTED** [review]
Some changes needed.

File: CHANGELOG.md (hunk 1 above)
Comment (CoMPaTech) [reply CHANGELOG.md:4]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (CoMPaTech) [reply CHANGELOG.md:8]: Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.4.0 - Release
+  - Improve error handling, group switching (Adam).
```
Comment (bouwew) [inline CHANGELOG.md:4]: One more change: add group switching

**Review by bouwew — APPROVED** [review]
All good!

**Pull request merged**